src/activity_actor.cpp

```diff
@@ -2384,6 +2384,8 @@ void insert_item_activity_actor::finish( player_activity &act, Character &who )
                 success = holster->put_in( it, item_pocket::pocket_type::CONTAINER,
                                            /*unseal_pockets=*/true ).success();
                 if( success ) {
+                    add_msg( string_format( _( "You insert your %s into the %s." ),
```

Comment:
```suggestion
                    who.add_msg_if_player( string_format( _( "You insert your %s into the %s." ),
```

Messages for things the player does (typically any message starting with the word "You") should use the more specific `add_msg_if_player` method. The `add_msg` method is unconditional and might conceivably be displayed when an NPC or monster does the activity. I know this file is already filled with incorrect usages of both functions, but it would be best not to make it worse :-)

Reply:
ah ok then i can skip the avatar check in the if